Tribler/Main/vwxGUI/settingsDialog.py

```diff
@@ -207,6 +209,19 @@ def saveAll(self, event):
             self.utility.write_config('use_webui', useWebUI)
             restart = True
 
+
+        becomeExitNode = self._become_exitnode.IsChecked()
+        if becomeExitNode != scfg.get_tunnel_community_exitnode_enabled():
+            scfg.set_tunnel_community_exitnode_enabled(becomeExitNode)
+            self.saveDefaultSessionConfig(scfg)
+            restart = True
+
+        switchHsOnTimeout = self._switch_hs_timeout.IsChecked()
+        if switchHsOnTimeout != scfg.get_tunnel_community_hs_timeout_switch():
+            scfg.set_tunnel_community_hs_timeout_switch(switchHsOnTimeout)
+            self.saveDefaultSessionConfig(scfg)
```

Comment:
There's no need for this, the scfg is saved lateron, and also remove the saveDefaultSessionConfg method.